message/abandon.go

```diff
@@ -15,8 +16,12 @@ type abandon struct {
 }
 
 func (a *abandon) Do(ctx context.Context, _ Handler, message *servicebus.Message) Handler {
+	ctx, span := startSpanFromMessageAndContext(ctx, "go-shuttle.abandon.Do", message)
+	defer span.End()
+
 	if err := message.Abandon(ctx); err != nil {
-		// trace the failure to abandon the message.
+		tab.For(ctx).Debug(err.Error())
```

Comment:
Why debug? Why not Error(err.Error()) warning.

Same for complete below.

Reply:
changed to Error